packages/react-magma-dom/src/components/Combobox/MultiCombobox.tsx

```diff
@@ -367,7 +378,7 @@ export function MultiCombobox<T>(props: MultiComboboxProps<T>) {
           ...getInputProps({
             ...options,
             ...getDropdownProps({
-              onKeyDown: onInputKeyDown,
+              onKeyDown: onInputKeyDown ? onInputKeyDown : handleEscape,
```

Comment:
Why is this mutually exclusive? Or should we always call both `onInputKeyDown` AND `handleEscape`? _same comment for single combobox_

Reply:
This question is still pending